components/compliance-service/dao/pgdb/migration/sql/41_create_telemetry.up.sql

```diff
@@ -0,0 +1,8 @@
+-- create table telemetry to store the last complaiance telemtry reported date
+CREATE TABLE IF NOT EXISTS telemetry (
+  id                         text PRIMARY KEY,
+  last_telemetry_reported_at TIMESTAMPTZ NOT NULL,
+  created_at                 TIMESTAMPTZ NOT NULL,
+  updated_at                 TIMESTAMPTZ NOT NULL,
```

Comment:
This field is not necessary. This should be similar to `last_telemetry_reported_at`.